### PR TITLE
Corrected "sidechain" to "child chain"

### PR DESCRIPTION
**Pull request by jet86**

### CODE REVIEW

**Comment by troggy** [comment]
@jet86 Tim, how do you define the difference between these terms?

**Comment by jet86** [comment]
The basic difference between side-chains and Plasma child chains is where the security is ultimately rooted.

> a sidechain relies on its own validation process to determine the security of that chain, whereas a Plasma child chain may have its own validation process too (such as a PoS layer) but its security ultimately depends on the validation of its root chain (Ethereum).
[Source](https://www.reddit.com/r/ethereum/comments/9hh68x/the_state_of_plasma_consensys_media/e6cypgi/?context=3)

The [Plasma whitepaper](https://plasma.io/plasma.pdf) itself also points out the difference (see for example section 4.3), as does section 7 of [this ELI5 version of the whitepaper](https://medium.com/@tousthilagavathy/eli5-of-plasma-whitepaper-72e7570829a).
